### PR TITLE
Added a more verbose warning message when deleting CS via the admin GUI.

### LINKED ISSUE #90: Inform user about destructive operation side-effects
After removing Compute Site from Atmosphere, VMTs located on that Compute Site are removed. I guess that flag `managed_by_atmosphere` decided if the VMT will be removed or not. But there is no information displayed for admin about side-effects of removing Compute Site.

/cc @nowakowski @mkasztelnik @Nuanda what are you thinking about informing admin about side-effects of some operation performed via Atmosphere?

### PR DESCRIPTION
**Pull request by nowakowski**
@mkasztelnik @paoolo Please review.

Fixes #90

### CODE REVIEW

**Comment by mkasztelnik** [comment]
@nowakowski should I tag it as ready for review?

File: config/locales/en.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ en:
   keep_it_secret: Keep it secret, keep it safe!
   reset: Reset
   are_you_sure: Are you sure?
+  are_you_sure_cs: Warning! This will delete all Virtual Machine Templates which exist on this CS. Are you sure?
```
Comment (paoolo) [inline config/locales/en.yml]: May you say "which exist on this CS and are managed by Atmosphere"? But I guess that safer is this what you proposed.

**Comment by mkasztelnik** [comment]
IMHO this is not enough - also virtual machines and flavors will be deleted. What is more note about managed by atmosphere would also be useful.

**Comment by nowakowski** [comment]
@mkasztelnik Sorry, I forgot about the tags.

**Comment by nowakowski** [comment]
@mkasztelnik There you go.

**Comment by mkasztelnik** [comment]
:+1:

**Comment by Nuanda** [comment]
:+1:

**Pull request merged**